auth-service/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  auth-api:
+    build: .
+    env_file:
+      - .env
+    ports:
+      - 8080:8080
+    volumes:
+      - .:/usr/src/app
+    command: go run cmd/main.go -b 0.0.0.0
+  db:
+    image: postgres:alpine
```

Comment:
suggest using this image for postgresql
https://hub.docker.com/r/bitnami/postgresql